apps/src/turtle/startBlocks.xml.ejs

```diff
@@ -324,16 +324,28 @@ var spiral = function(i) {
     <% //  Define "draw a triangle". -%>
     <%- polygon({
       title: msg.drawATriangle(),
-      modifiers: (level == 6 ? 'x="220" y="250"' : 'x="220" y="250" editable="false" deletable="false"'),
+      modifiers: (level == 8 ? '' : 'x="220" y="250"') + (level == 6 ? '' : ' editable="false" deletable="false"'),
       sides: 3,
       length: (level >= 7 ? 0 : 100)
     }) -%>
   <% } -%>
-  <% if (level == 7) { -%>
+  <% if (level == 8 ) { -%>
+    <block type="procedures_callnoreturn" inline="false">
+      <mutation name="<%= msg.drawAHouse() %>">
+        <arg name="<%= msg.lengthParameter() %>"></arg>
+      </mutation>
+        <value name="ARG0">
+          <block type="math_number">
+            <title name="NUM">100</title>
+          </block>
+        </value>
+    </block>
+  <% } -%>
+  <% if (level == 7 || level == 8) { -%>
     <% //  Define "draw a house". -%>
-    <block type="procedures_defnoreturn" x="220" y="460">
+    <block type="procedures_defnoreturn" <% if (level == 7) { -%>x="220" y="460"<% } -%>>
       <mutation>
-        <% if (level == 11) { %><arg name="<%= msg.lengthParameter() %>"></arg><% } -%>
+        <% if (level == 8) { %><arg name="<%= msg.lengthParameter() %>"></arg><% } -%>
```

Comment:
Yep!  This was inside an `<% if (level == 7) { -%>` block so it was unreachable code.